### PR TITLE
#535 create reference sections for l gs

### PR DESCRIPTION
**Pull request by gernotstarke**

### CODE REVIEW

File: docs/04-documentation/LG-04-09.adoc [hunk 1]
```diff
@@ -6,8 +6,8 @@
 Softwarearchitekt:innen kennen:
 
 * Grundlagen mehrerer publizierter Frameworks zur Beschreibung von Softwarearchitekturen, beispielsweise:
-** ISO/IEC/IEEE 42010, siehe <<iso42010>>
-** arc42, siehe <<arc42>>
+** ISO/IEC/IEEE 42010, 
+** arc42,
 ** C4, siehe <<brownc4>>
 ** FMC, siehe <<fmc>>
```
Comment (sippsack) [inline docs/04-documentation/LG-04-09.adoc:12]: same as in LG 04-03: why are brownc4 and fmc still referenced?

File: docs/04-documentation/LG-04-09.adoc [hunk 2]
```diff
@@ -22,11 +22,15 @@ Softwarearchitekt:innen kennen:
 Software architects know:
 
 * basics of several published frameworks for the description of software architectures, for example:
-** ISO/IEC/IEEE 42010, see <<iso42010>>
-** arc42, see <<arc42>>
+** ISO/IEC/IEEE 42010, 
+** arc42,
 ** C4, see <<brownc4>>
 ** FMC, see <<fmc>>
```
Comment (sippsack) [inline docs/04-documentation/LG-04-09.adoc:28]: To be removed: brownc4 and fmc

**Review by sippsack — CHANGES_REQUESTED** [review]

File: docs/04-documentation/LG-04-03.adoc [hunk 3]
```diff
@@ -12,8 +12,8 @@ Softwarearchitekt:innen kennen mindestens folgende UML-Diagramme (siehe <<uml>>)
 
 Softwarearchitekt:innen kennen Alternativen zu UML, beispielsweise (R3)
 
-* ArchiMate, siehe <<archimate>> 
-* SysML, siehe <<sysml>>
+* ArchiMate, 
+* SysML,
```
[the remark this answers is not in the document]
Comment (gernotstarke) [reply docs/04-documentation/LG-04-03.adoc:16]: removed commas

File: docs/04-documentation/LG-04-03.adoc [hunk 4]
```diff
@@ -33,11 +33,15 @@ Software architects know at least the following UML (see <<uml>>) diagrams to de
 
 Software architects know alternative notations to UML diagrams, for example: (R3)
 
-* Archimate, see <<archimate>>
-* SysML, see <<sysml>>
+* Archimate,
+* SysML,
 * Entity-relationship diagrams, see <<erd>>
 * C4, see <<brownc4>>
```
[the remark this answers is not in the document]
Comment (gernotstarke) [reply docs/04-documentation/LG-04-03.adoc:39]: The reason is that it is not obvious from the author's names what part of the content the reference refers to.

Some readers for example might not know _Simon Brown_ - and therefore cannot relate Brown-94 to the C4-model mentioned in the learning goal. 

Therefore I propose to keep the specific references in a few cases.

@ulibecker proposed to defer that to the glossary - which could be a great option, IF we had adequate glossary entries.

File: docs/04-documentation/LG-04-09.adoc (hunk 1 above)
Comment (gernotstarke) [reply docs/04-documentation/LG-04-09.adoc:12]: see above.

File: docs/04-documentation/LG-04-09.adoc (hunk 2 above)
Comment (gernotstarke) [reply docs/04-documentation/LG-04-09.adoc:28]: see above.

**Review by programming-wolf — DISMISSED** [review]

**Pull request merged**